src/domain/metric/service/postMetric.js

```diff
@@ -30,7 +31,7 @@ export default (posts: Post[]): PostMetric[] => {
     return {
       post,
       id: post.id,
-      createdTime: new Date(post.created_time),
+      createdTime: moment(post.created_time).format(),
```

Comment:
@reyhansofian this should be `Date` since `moment` is not standard, modify on other other files if need `moment` based.